lists2safebrowsing.py

```diff
@@ -690,7 +704,7 @@ def main():
     if log_file:
         log_file.close()
 
-    publish_to_cloud(config, chunknum)
+    # publish_to_cloud(config, chunknum)
```

Comment:
Is this intentional?

Reply:
Corrected [44dba11](https://github.com/mozilla-services/shavar-list-creation/pull/176/commits/44dba11156d7ed7422827d3f7b75394f9ceaed92)